translationtable/ebi.yaml

```diff
@@ -0,0 +1,123 @@
+# See https://www.clinicalgenome.org/site/assets/files/2757/fitzpatrick_ddg2p.pdf
```

Comment:
Want name to local tt name to be a common function of ingest name it is local to